CIP-0062/README.md

```diff
@@ -126,32 +126,9 @@ The vote options. This is the set of options we can vote for.
 
 # **`Delegation API`**
 
-## **api.getVotingKeys**(): Promise<cbor<PublicKey\>[]>
-Should return a list of all the voting keys for the current wallet. 
+## **api.getVotingKey**(): Promise\<cbor<PublicKey\>>
```

Comment:
what should this call return if there's no voting key registered yet?

Reply:
also, as per the CDDL, a user may register multiple voting keys in a single registration, and multiple of them may be theirs, in theory. In such case, I'd expect this call to return a list of voting keys, or is that case impossible/against specs?

Reply:
> what should this call return if there's no voting key registered yet? I'd expect it to return null but the specs apparently don't mention that

ok depends on the definition of "in use", whether it means "registered" or "the next in line to be registered" or something else. But I'd expect to have a call that would be able to provide both keys - the latest registered voting key makes sense to be able to show voting stats/construct the voting transaction and the "the next in line to be registered" key makes sense for the dapp to build the voting delegation. I'd therefore expect actually two different calls to be able to retrieve any of these two keys, as needed, but maybe I'm missing something

Reply:
@refi93 The latest iteration of this spec, expects a single key to be created / "in-use" . We discussed supporting a N keys to be created, but for now seems unnecessary. Doesn't mean we wouldn't expand it in the future. 

Does it makes sense?

Reply:
makes sense :+1: